model/jpa/src/main/java/org/keycloak/models/jpa/JpaRealmProvider.java

```diff
@@ -595,7 +603,7 @@ public Stream<GroupModel> getTopLevelGroupsStream(RealmModel realm) {
 
     @Override
     public Stream<GroupModel> getTopLevelGroupsStream(RealmModel realm, Integer first, Integer max) {
```

Comment:
Isn't possible to just forward the method call to the new `getTopLevelGroupsStream(RealmModel realm, String search, Integer firstResult, Integer maxResults`) method and remove its block entirely (looks like a duplicate now)?

Reply:
addressed this in the current commit